verification.py

```
@@ -1,27 +1,55 @@
+    print("Verification complete!")
 
 if __name__ == "__main__":
+    os.makedirs("verification/video", exist_ok=True)
     with sync_playwright() as p:
         browser = p.chromium.launch(headless=True)
-        context = browser.new_context(record_video_dir="verification/video")
+        context = browser.new_context(record_video_dir="verification/video", viewport={"width": 1280, "height": 720})
         page = context.new_page()
         try:
             verify_feature(page)
+        except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)